### DISCUSSION TITLE
No module motifs

### DISCUSSION THREAD

**Discussion by siebert-julien** [TDAmeritrade/stumpy#420] [Help: Coding & Implementations] [answered]
Hello,

I am a little bit perplexed. I do not know if I forgot to install something.
I have stumpy v1.8.0 installed on Windows 10, Python 3.8.6.
I installed stumpy via pip (I did not try the install directly from the source code, yet) in my virtual environment.

It seems that there is no module motifs available.

I am trying to call this piece of code:

```python
from stumpy import motifs
```

I get the following error:
```
AttributeError: module 'stumpy' has no attribute 'motifs'
```

I also checked the files installed under site-packages/stumpy and there is no file motifs.py

Am I missing something here?

Best regards, Julien

**Comment by seanlaw** [comment, marked as answer]
@siebert-julien Welcome to the STUMPY community and thank you for your question. Currently, the `motifs` module is set to be released in the upcoming v1.9.0 release though, if you install STUMPY from source then you can access it now. Unfortunately, our docs follow the development branch and so some features may be described in the API docs prior to release. Sorry for the confusion.

**Reply by siebert-julien** [reply]
No problem, thanks for the prompt answer. I'll install from source then.

**Reply by seanlaw** [reply]
> No problem, thanks for the prompt answer. I'll install from source then. 

Please let me know if you run into any issues with the installation.
